requestor/erigolem_cli.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python3
```

Comment:
It assumes activation the venv in the environment. Probably better is to skip activation and call directly:
```suggestion
#!./.venv/bin/python
```
I'm not sure it will work when called outside of the `requestor` dir 🤔 

- if it is easy to make it executable from any dir let's do it. 
- otherwise let's assume it'll ever be called from the requestor dir 👐

Reply:
No, we cannot to assume it will only called from within virtualenv. I'm just putting it in a Docker image where there will be no venv. I refuse to make this change.

Reply:
Well, I'm not insisting, so you can reject the whole idea. What I meant is setup (venv activation) and usage is/might be separated. I don't want you to fix python :)